README.md

```diff
@@ -79,15 +79,25 @@ If you click on the icon which looks like a set of gears, you should see an inte
 
 ![Rhasspy settings](rhasspy/images/Rhasspy_settings.PNG)
 
-It is recommended that you set all of the settings to the same thing as pictured above, but you may use whichever text to speech option you like. You will also have to fill out the device information underneath the Audio Recording and Audio Playing settings. You can also choose whichever wake word you prefer under Wake Word. When you're done, click on the Save Settings button. Please note that you may have to do this iteratively, and you may be asked to download data for some of the settings. You can test the audio recording and playback by clicking on the icon that looks like a house.
+It is recommended that you set all of the settings to the same thing as pictured above, but you may use whichever text to speech option you like. You can also choose whichever wake word you prefer under Wake Word. When you're done, click on the Save Settings button. Please note that you may have to do this iteratively, and you may be asked to download data for some of the settings. A download button should appear at the top of the page like below if aditional data is needed.
+
+![image](https://github.com/wyattschool/sandman/assets/77877259/505852bd-9a8d-4181-b9b4-bb2e95cb26bb)
+
+You will also have to fill out the device information underneath the Audio Recording and Audio Playing settings. Clcik the green colored area with the carret to the left to expose the microphone/Audio Recording or speaker/Audio Playing settings. Once expanded you will need to select the correct available devices. For instance the audio recording device might be "Default Audio Device (sysdefault:CARD=Device_1)" and the audio playing device might be "Default Audio Device (sysdefault:CARD=Device)". It might be helpful to plug in one device at a time and review which device has been added to the list. The list is long so this may take some trial and error. 
```

Comment:
Typo: Clcik->Click. Don't you also have to press the refresh button before the list is populated? I wonder if the ordering of this should be plug-in speaker, try to select the speaker device, test, then repeat with microphone?

Reply:
The refresh is required you are correct. I agree the order you describe of doing it may work and could be better but I also think that the way I describe is valid. I am willing to change the ordering if that is desired.